packages/ia-pic-uploader/src/ia-pic-uploader.ts

```diff
@@ -0,0 +1,834 @@
+          class="drag-text ${this.showLoadingIndicator ? 'pointer-none' : ''}"
+          @keyup=""
+          @click=${() => {
+            this.dropRegion?.click();
+          }}
+          >${this.taskStatus
+            ? this.taskStatus
+            : 'Drag & Drop an image file here or'}</span
+        >
+        <button
+          id="file-picker"
+          @click=${() => {
+            this.dropRegion?.click();
+          }}
+          class="ia-button primary ${this.showLoadingIndicator
+            ? 'pointer-none hidden'
+            : ''}"
+        >
+          Pick image to upload
+        </button>
+        <div
+          class="validationErrorDiv"
+          style="display:${this.fileValidationError === '' ? 'none' : 'block'}"
+        >
+          <span class="fileValidationError">${this.fileValidationError}</span>
+        </div>
+        <form
+          method="post"
+          id="save-file"
+          enctype="multipart/form-data"
+          action="${formAction}"
+        >
+          <input
+            class="file-selector"
+            name="file"
+            type="file"
+            accept="image/*"
+            style="display: none;"
+          />
+          <input type="hidden" name="identifier" .value="${this.identifier}" />
+          <button
+            id="file-submit"
+            type="submit"
+            name="submit"
+            value="SUBMIT"
+            class="ia-button
+            ${!this.showDropper || this.fileValidationError !== ''
+              ? 'hidden'
+              : ''}
+            ${this.fileValidationError || this.showLoadingIndicator
+              ? 'pointer-none hidden'
+              : ''}"
+          >
+            ${this.showLoadingIndicator
+              ? this.loadingIndicatorTemplate
+              : 'Submit'}
+          </button>
+        </form>
+        <div
+          class="image-preview full-preview 
+          ${this.showLoadingIndicator ? 'pointer-none hidden' : ''}"
+          @keyup=""
+          @click=${() => {
+            this.dropRegion?.click();
+          }}
+        ></div>
+      </div>
+    `;
+  }
+
+  /**
+   * function that render html template for compact version
+   * @returns {HTMLElement}
+   */
+  get getSelectFileTemplate() {
+    return html`
+      <div class="select-region">
+        <input
+          class="file-selector"
+          name="file"
+          type="file"
+          accept="image/*"
+          style="display: none;"
+        />
+        <div class="select-message">
+          Drop a new image onto<br />your picture here or<br />
+          <a
+            href="#"
+            id="upload-region"
+            class="${this.showLoadingIndicator ? 'pointer-none' : ''}"
+            >select an image to upload</a
+          >
+        </div>
+      </div>
+    `;
+  }
+
+  /**
+   * function that render html for overlay form compact version
+   * @returns {HTMLElement}
+   */
+  get getOverlayIcon() {
+    return html`
+      <div
```

Comment:
Suggestion: `get selectFileTemplate()` and `get overlayIcon()`? I think the general practice is to omit `get` from the name of JS/TS getters.

Reply:
resolved